src/ensRegistry.ts

```diff
@@ -87,12 +102,7 @@ function _handleNewResolver(event: NewResolverEvent, isMigrated:boolean): void {
   let id = event.params.resolver.toHexString().concat('-').concat(event.params.node.toHexString())
 
   let node = event.params.node.toHexString()
-  let domain = new Domain(node)
-  if(node == ROOT_NODE){
-    domain.owner = EMPTY_ADDRESS
-    domain.isMigrated = isMigrated
```

Comment:
You can get rid of `isMigrated:boolean` from `_handleNewResolver` as `isMigrated` variable is no longer in use